src/main/scala/com/comcast/xfinity/sirius/uberstore/segmented/SegmentedCompactor.scala

```diff
@@ -15,15 +15,28 @@
  */
 package com.comcast.xfinity.sirius.uberstore.segmented
 
+import com.comcast.xfinity.sirius.api.SiriusConfiguration
+import com.comcast.xfinity.sirius.api.impl.{Delete, OrderedEvent}
+
 import scalax.file.Path
 import java.io.File
 import annotation.tailrec
 
 object SegmentedCompactor {
-
   val COMPACTING_SUFFIX = ".compacting"
   val TEMP_SUFFIX = ".temp"
 
+  def apply(siriusConfig: SiriusConfiguration) : SegmentedCompactor = {
+
+    val maxDeleteAgeHours = siriusConfig.getProp(SiriusConfiguration.COMPACTION_MAX_DELETE_AGE_HOURS, Integer.MAX_VALUE) // a really long time by default
```

Comment:
Please remove the trailing comment, I think it goes over the 120 char max width. It can either go on the line above or it can just be removed, I think it's fine either way.